federatedrec/factorization_machine/README.md

```diff
@@ -0,0 +1,34 @@
+# Federated Factorization Machine
+
+Factorization Machine(FM) is a  supervised learning approach incorporates second-order feature interaction.
+Federated factorization machine computes these cross-party cross-features and their gradients under encryption. 
+
+Here we simplify participants of the federation process into three parties. Party A represents Guest, party B represents Host. Party C, which is also known as “Arbiter,” is a third party that works as coordinator. Party C is responsible for generating private and public keys.
+
+## Heterogeneous FM
```

Comment:
The readme here can't show the federated processing, can some references be attached to here ot the image can show more ? Like encryption and gradient exchanges

Reply:
Thanks for remind. Will check with the author, and update here as soon as possible.